Assess the health of external test group

For each of the tests listed in https://github.com/adoptium/aqa-tests/tree/master/external we should assess their current health and status for all versions and implementations.

- [Hotspot spreadsheet](https://docs.google.com/spreadsheets/d/1kcHs1f7H1bmlLvkRBZ0CMdTaGBa3DWEXP86bYRefxg8/edit?usp=sharing)
- [OpenJ9 spreadsheet](https://docs.google.com/spreadsheets/d/197JBPA9hyYDLwc7xNncUKoXiKj9B7BH2Itx8ktdYgE4/edit?usp=sharing)

11: sanity.external https://ci.adoptopenjdk.net/view/work-in-progress/job/grinder_sandbox_new/440/console
openliberty_microprofile_tck failed. Issue raised https://github.com/adoptium/aqa-tests/issues/3380

```
13:15:04  16 errors
13:15:04  
13:15:04  FAILURE: Build failed with an exception.
13:15:04  
13:15:04  * What went wrong:
13:15:04  Execution failed for task ':com.ibm.ws.componenttest:compileJava'.
13:15:04  > Compilation failed; see the compiler error output for details.
13:15:04  
13:15:04  * Try:
13:15:04  Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
```

17: sanity.external https://ci.adoptopenjdk.net/view/work-in-progress/job/grinder_sandbox_new/442/console

openliberty_microprofile_tck_0 failed. Issue raised https://github.com/adoptium/aqa-tests/issues/3381

```
20:00:41  Picked up JAVA_TOOL_OPTIONS: -Dfile.encoding=UTF8 -Djava.security.egd=file:/dev/./urandom
20:00:57  java.lang.NoClassDefFoundError: Could not initialize class org.codehaus.groovy.vmplugin.v7.Java7
20:00:57  	at org.codehaus.groovy.vmplugin.VMPluginFactory.<clinit>(VMPluginFactory.java:43)
20:00:57  	at org.codehaus.groovy.reflection.GroovyClassValueFactory.<clinit>(GroovyClassValueFactory.java:35)
20:00:57  	at org.codehaus.groovy.reflection.ClassInfo.<clinit>(ClassInfo.java:107)
20:00:57  	at org.codehaus.groovy.reflection.ReflectionCache.getCachedClass(ReflectionCache.java:95)
20:00:57  	at org.codehaus.groovy.reflection.ReflectionCache.<clinit>(ReflectionCache.java:39)
20:00:57  	at org.codehaus.groovy.runtime.metaclass.MetaClassRegistryImpl.registerMethods(MetaClassRegistryImpl.java:210)
...
20:00:57  FAILURE: Build failed with an exception.
20:00:57  
20:00:57  * What went wrong:
20:00:57  Could not initialize class org.codehaus.groovy.reflection.ReflectionCache
20:00:57  
20:00:57  * Try:
20:00:57  Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
20:00:57  
20:00:57  * Get more help at https://help.gradle.org/
20:00:57  
20:00:57  BUILD FAILED in 14s
20:00:57  
20:00:57  openliberty_microprofile_tck_0_FAILED
```

quarkus_quickstarts_test_0 failed. Issue raised https://github.com/adoptium/aqa-tests/issues/3382

```
20:00:35  /quarkus-quickstarts
20:00:35  Compile and run quarkus_quickstarts tests
20:00:35  Picked up JAVA_TOOL_OPTIONS: -Dfile.encoding=UTF8 -Djava.security.egd=file:/dev/./urandom
20:00:38  [[1;31mERROR[m] Error executing Maven.
20:00:38  [[1;31mERROR[m] java.lang.IllegalStateException: Unable to load cache item
20:00:38  [[1;31mERROR[m] Caused by: Unable to load cache item
20:00:38  [[1;31mERROR[m] Caused by: Could not initialize class com.google.inject.internal.cglib.core.$MethodWrapper
20:00:38  Build quarkus_quickstarts completed
```

--- a/external/external_custom/build.xml
+++ b/external/external_custom/build.xml
@@ -9,8 +9,10 @@
 	<!-- set properties for this build -->
 	<property name="TEST" value="external_custom" />
 	<property environment="env" />
-	<property name="EXTERNAL_CUSTOM_REPO" value="${env.EXTERNAL_CUSTOM_REPO}" />
-    <condition property="EXTERNAL_REPO_BRANCH" value="${env.EXTERNAL_REPO_BRANCH}" else="master">
+	<condition property="EXTERNAL_CUSTOM_REPO" value="${env.EXTERNAL_CUSTOM_REPO}" else="">
+		<isset property="env.EXTERNAL_CUSTOM_REPO" />
+	</condition>
+	<condition property="EXTERNAL_REPO_BRANCH" value="${env.EXTERNAL_REPO_BRANCH}" else="master">
 		<isset property="env.EXTERNAL_REPO_BRANCH" />
 	</condition>
 	<condition property="EXTERNAL_TEST_CMD" value="${env.EXTERNAL_TEST_CMD}" else="mvn clean install">
@@ -30,13 +32,9 @@
 	</target>
 
 	<target name="build">
-		<!-- Temporarily disable for openj9 and ibm due to https://github.com/adoptium/aqa-tests/issues/2903 -->
 		<if>
 			<not>
-				<or>
-					<equals arg1="${JDK_IMPL}" arg2="ibm"  />
-					<equals arg1="${JDK_IMPL}" arg2="openj9" />
-				</or>
+				<equals arg1="${EXTERNAL_CUSTOM_REPO}" arg2=""/>
 			</not>
 			<then>
 				<antcall target="dist" inheritall="true" />

--- a/external/external_custom/playlist.xml
+++ b/external/external_custom/playlist.xml
@@ -21,7 +21,7 @@
 		$(TEST_ROOT)$(D)external$(D)external.sh --clean --tag ${DOCKERIMAGE_TAG} --version ${JDK_VERSION} --impl ${JDK_IMPL} --dir external_custom
 		</command>
 		<levels>
-			<level>extended</level>
+			<level>special</level>
 		</levels>
 		<groups>
 			<group>external</group>

